common/artifacts/000_wc_artifacts.txt

```diff
@@ -838,6 +1183,256 @@ prosthetic_blade_hand = {
 
 	picture = "GFX_inv_weapon_shortblade_29"
 }
+warsong_battle_drums = {
+	quality = 3
+	flags = { misc rare }
+
+	active = {
+		is_adult = yes
+		
+	}
+
+	indestructible = yes
+
+	martial = 1
+
+	monthly_character_prestige = 0.75
+	command_modifier = {
+#		actual_heavy_infantry = 0.05
+#		actual_light_infantry = 0.05
+#		archers = 0.05
+#		light_cavalry = 0.05
+		offence = 0.1
+		morale_offence = 0.1
+	}
+	picture = "GFX_warsong_battle_drums"
+	
+	slot = leader
+}
+scourge_bone_chimes = {
+	quality = 3
+	flags = { misc rare }
+
+	active = {
+		is_adult = yes	
+			OR = {
+			is_death_knight_class_trigger = yes
+			is_necromancer_class_trigger = yes
+			is_warlock_class_trigger = yes
+			OR = {
+				religion = death_god
+				religion_group = fel_group
+			}
+		}
+	}
+
+	indestructible = yes
+
+	martial = 1
+
+	monthly_character_prestige = 0.75
+
+	general_opinion = -5
+	death_god_opinion = 5
+	command_modifier = {
+		morale_defence = 0.1
+		morale_offence = 0.1
+	}
+	picture = "GFX_scourge_bone_chimes"
+	
+	slot = leader
+}
+frost_wyrm_skull_shield = {
+	quality = 3
+	flags = { misc epic }
+
+	active = {
+		is_adult = yes
+	}
+
+	indestructible = yes
+
+	martial = 2
+
+	combat_rating = 20
+
+	monthly_character_prestige = 0.75
+	
+	command_modifier = {
+		morale_defence = 0.05
+		defence = 0.05
+	}
+
+	picture = "GFX_frost_wyrm_skull_shield"
+	slot = libram
+}
+orb_of_kiljaeden = {
+	quality = 3
+	flags = { misc fel_artifact powerful_magical_artifact }
+
+	active = {
+		is_adult = yes
+		
+		use_dark_artifacts_trigger = yes
+	}
+	allowed_gift = {
+		always = no
+	}
+
+	stacking = no
+	indestructible = yes
+
+	martial = 1
+	intrigue = 2
+
+	combat_rating = 15
+
+	monthly_character_prestige = 0.45
+	monthly_character_piety = 0.15
+	
+	general_opinion = -10
+	fel_group_opinion = 15
+
+	picture = "GFX_orb_of_kiljaeden"
+	slot = libram
+}
+orb_of_corruption = {
+	quality = 3
+	flags = { misc epic }
+
+	active = {
+		is_adult = yes
+	}
+
+	indestructible = yes
+
+	martial = 2
+	intrigue = 2
+
+	combat_rating = 10
+
+	monthly_character_prestige = 0.75
+
+	general_opinion = -10
+	fel_group_opinion = 7
+	death_god_opinion = 8
+
+	picture = "GFX_orb_of_corruption"
+	slot = libram
+}
+lionhorn_of_stormwind = {
+	quality = 3
+	flags = { misc rare }
+
+	indestructible = yes
+
+	martial = 1
+
+	monthly_character_prestige = 1
+	
+	command_modifier = {
+		morale_offence = 0.1
+		morale_defence = 0.1
+	}
+	picture = "GFX_lionhorn"
+
+	slot = leader
+}
+spellbook_of_drekthar = {
+	quality = 3
+	flags = { misc epic }
+
+	active = {
+		is_adult = yes			# maybe reduce requirements for shaman education?
+	
+		is_magic_class_trigger = yes
```

Comment:
I would remove it. Imagine, you're smart guy (more then 12 Learning), but want to become mage. You need +3 Learning to do it. You would get it from this book, but you can't use it because you're not mage yet. So, you need this book, that teach magic, to become mage... but to read this book you need to be mage.

Reply:
done